storage/innobase/log/log0recv.cc

```diff
@@ -1881,48 +1793,17 @@ static byte *recv_parse_or_apply_log_rec_body(
       if (!recv_recovery_on) {
         index_load_list.emplace_back(
             std::pair<space_id_t, lsn_t>(space_id, recv_sys->recovered_lsn));
->>>>>>> mysql-8.0.31
       }
+#endif /* UNIV_HOTBACKUP */
 
-      return fil_tablespace_redo_delete(
-          ptr, end_ptr, page_id_t(space_id, page_no), parsed_bytes,
-          recv_sys->bytes_to_ignore_before_checkpoint != 0 ||
-              recv_sys->recovered_lsn + parsed_bytes <
-                  backup_redo_log_flushed_lsn);
-
-    case MLOG_FILE_CREATE:
-
-      return fil_tablespace_redo_create(
-          ptr, end_ptr, page_id_t(space_id, page_no), parsed_bytes,
-          recv_sys->bytes_to_ignore_before_checkpoint != 0 ||
-              recv_sys->recovered_lsn + parsed_bytes <
-                  backup_redo_log_flushed_lsn);
-
-    case MLOG_FILE_RENAME:
-
-      return fil_tablespace_redo_rename(
-          ptr, end_ptr, page_id_t(space_id, page_no), parsed_bytes,
-          recv_sys->bytes_to_ignore_before_checkpoint != 0 ||
-              recv_sys->recovered_lsn + parsed_bytes <
-                  backup_redo_log_flushed_lsn);
-
-    case MLOG_FILE_EXTEND:
-
-      return fil_tablespace_redo_extend(
-          ptr, end_ptr, page_id_t(space_id, page_no), parsed_bytes,
-          recv_sys->bytes_to_ignore_before_checkpoint != 0 ||
-              recv_sys->recovered_lsn + parsed_bytes <
-                  backup_redo_log_flushed_lsn);
-#endif /* !UNIV_HOTBACKUP */
-    case MLOG_INDEX_LOAD:
-#if defined(UNIV_HOTBACKUP) || defined(XTRABACKUP)
+#ifdef XTRABACKUP
       /* While scaning redo logs during  backup phase a
       MLOG_INDEX_LOAD type redo log record indicates a DDL
       (create index, alter table...)is performed with
       'algorithm=inplace'. This redo log indicates that
 
-      1. The DDL was started after MEB started backing up, in which
-      case MEB will not be able to take a consistent backup and should
+      1. The DDL was started after PXB started backing up, in which
+      case PXB will not be able to take a consistent backup and should
```

Comment:
A few lines below, we still have mentions to MEB inside a solo `#ifdef XTRABACKUP` directive

Reply:
fixed